ql/src/test/results/clientpositive/llap/update_where_partitioned.q.out

```diff
@@ -63,15 +63,20 @@ PREHOOK: type: QUERY
 PREHOOK: Input: default@acid_uwp
 PREHOOK: Input: default@acid_uwp@ds=today
 PREHOOK: Input: default@acid_uwp@ds=tomorrow
+PREHOOK: Output: default@acid_uwp
 PREHOOK: Output: default@acid_uwp@ds=today
 PREHOOK: Output: default@acid_uwp@ds=tomorrow
 POSTHOOK: query: update acid_uwp set b = 'fred' where b = 'k17Am8uPHWk02cEf1jet'
 POSTHOOK: type: QUERY
 POSTHOOK: Input: default@acid_uwp
 POSTHOOK: Input: default@acid_uwp@ds=today
 POSTHOOK: Input: default@acid_uwp@ds=tomorrow
+POSTHOOK: Output: default@acid_uwp
+POSTHOOK: Output: default@acid_uwp@ds=today
 POSTHOOK: Output: default@acid_uwp@ds=today
 POSTHOOK: Output: default@acid_uwp@ds=tomorrow
+POSTHOOK: Lineage: acid_uwp PARTITION(ds=today).a SIMPLE [(acid_uwp)acid_uwp.FieldSchema(name:a, type:int, comment:null), ]
```

Comment:
w.r.t to security stuff; this kinda means that we will be "reading" column `A` in case of an update of column `B`; I think an interesting question would be:
should we support the case of creating ranger policies in which a user is allowed to update a column (b) in a table; but not allowed to read some other column (`a`) - I think right now we support this; should we keep doing that? I don't know how complicated would be to omit the columns we are touching only because of this rewrite